src/views/ProfilePage/Profile.js

```diff
@@ -0,0 +1,338 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Loader from 'react-loader-spinner';
+import Header from '@Components/Header/index';
+import Button from '@Components/Button/index';
+import Footer from '@Components/Footer/index';
+import Dialog from '@Components/Dialog/index';
+import Input from '@Components/Input/index';
+import TextArea from '@App/components/TextArea/TextArea';
+import ArticleCard from '@Components/ArticleCard/index';
```

Comment:
Please when you are import do not do this `import Dialog from '@Components/Dialog/index';` or 
`import TextArea from '@App/components/TextArea/TextArea';` I mentioned this before.

You import like this `import TextArea from '@App/components/TextArea';` the index file is automatically called which calls the `TextArea.js` file.

Please I don't want to make mention of this again to the entire team.

Reply:
noted, I will make the corrections now

Reply:
Did you make the necessary changes? @chiboycalix

Reply:
yes I did.